scripts/functional-tests.sh

```diff
@@ -330,7 +284,7 @@ test_loading_session_history(){
     '
 }
 
-test_generate(){
+test_data_generate(){
```

Comment:
For instance, `test_server_welcome_message‎` could be `test_model_server_welcome_message‎`? Honestly, the rename is not making this more clear or anything, but it might cause issues for backports. If we are not planning to backport this I'm ok keep it. Thanks!

Reply:
Definitely won't be backported! And I see your point, I think we can figure out a consistent name scheme in https://github.com/instructlab/instructlab/issues/748?